fdbbackup/backup.actor.cpp

```diff
@@ -3304,6 +3460,8 @@ int main(int argc, char* argv[]) {
 	int status = FDB_EXIT_SUCCESS;
 
 	std::string commandLine;
+	std::string subCommand;
```

Comment:
Make new two arrays: 1. for options and their parameters and 2. for words that are not options